official/wasm/cmd/cmd.go

```diff
@@ -0,0 +1,127 @@
+package cmd
+
+import (
+	"os"
+	"path/filepath"
+	"sort"
+	"strings"
+
+	"github.com/ignite/cli/v28/ignite/pkg/cliui/colors"
+	"github.com/ignite/cli/v28/ignite/pkg/xgenny"
+	"github.com/ignite/cli/v28/ignite/services/chain"
+	"github.com/spf13/cobra"
+)
+
+// NewWasm creates a new wasm command that holds
+// some other sub commands related to CosmWasm.
+func NewWasm() *cobra.Command {
+	c := &cobra.Command{
+		Use:           "wasm [command]",
+		Aliases:       []string{"w"},
+		Short:         "Ignite wasm integration",
+		SilenceUsage:  true,
+		SilenceErrors: true,
+	}
+
+	// add sub commands.
+	c.AddCommand(
+		NewWasmAdd(),
+		NewWasmConfig(),
+	)
+	return c
+}
+
+const (
+	flagPath = "path"
+	flagHome = "home"
+
+	statusScaffolding  = "Scaffolding..."
+	statusAddingConfig = "Adding config..."
+)
+
+var (
+	modifyPrefix = colors.Modified("modify ")
+	createPrefix = colors.Success("create ")
+	removePrefix = func(s string) string {
+		return strings.TrimPrefix(strings.TrimPrefix(s, modifyPrefix), createPrefix)
+	}
+)
+
+func flagSetPath(cmd *cobra.Command) {
+	cmd.Flags().StringP(flagPath, "p", ".", "path of the app")
+}
+
+func flagSetHome(cmd *cobra.Command) {
+	cmd.Flags().String(flagHome, "", "directory where the blockchain node is initialized")
+}
+
+func getPath(cmd *cobra.Command) string {
+	path, _ := cmd.Flags().GetString(flagPath)
+	return path
+}
+
+func getHome(cmd *cobra.Command) string {
+	home, _ := cmd.Flags().GetString(flagHome)
+	return home
+}
+
+// newChainWithHomeFlags create new *chain.Chain with home and path flags.
+func newChainWithHomeFlags(cmd *cobra.Command, chainOption ...chain.Option) (*chain.Chain, error) {
```

Comment:
Maybe we could make this function public in the CLI, if it makes sense

Reply:
Agree, but IMHO wee need to change a little bit the command to not export from the `cmd` package